fleet-ms/src/main/liberty/config/server.xml

```diff
@@ -25,6 +25,6 @@
   <!-- Automatically expand WAR files and EAR files -->
   <applicationManager autoExpand="true"/>
 
-  <webApplication name="fleetms" location="${app.location}"/>
+  <webApplication name="fleetms" location="fleetms.war"/>
```

Comment:
@josiemundi I have a feeling that this is not working properly.  I just started the app using the latest image and I'm seeing this in the logs:
```
CWWKZ0014W: The application fleetms could not be started as it could not be found at location fleetms.war.
```
It does not seem to be serving any of the /fleetms routes successfully.

Reply:
Looks like the war file is being built with a different name to what was expected here:
```
sh-4.4$ find . -name "fleetms*"
./usr/servers/defaultServer/apps/fleetms-1.0-SNAPSHOT.war
```